web/src/features/panels/search-panel/getSearchData.ts

```diff
@@ -0,0 +1,41 @@
+import { t } from 'i18next';
+import { GridState } from 'types';
+
+import { ZoneRowType } from './ZoneList';
+
+export const getAllZones = () => {
+  // Get all zone data directly from translations
+  const zoneData = t('zoneShortName', { returnObjects: true }) as Record<
+    string,
+    ZoneRowType
+  >;
+
+  return zoneData;
+};
+
+export const getFilteredList = (
+  searchTerm: string,
+  zoneData: Record<string, ZoneRowType>
+): ZoneRowType[] => {
+  if (!searchTerm) {
+    return [];
+  }
+
+  // Filter zones based on search term
+  const filtered = Object.entries(zoneData).filter(([zoneKey, zone]) => {
+    const searchLower = searchTerm.toLowerCase();
+    return (
+      zoneKey.toString().toLowerCase().includes(searchLower) ||
+      zone.zoneName?.toLowerCase().includes(searchLower) ||
+      zone.countryName?.toLowerCase().includes(searchLower) ||
+      zone.seoZoneName?.toLowerCase().includes(searchLower) ||
+      zone.displayName?.toLowerCase().includes(searchLower)
+    );
+  });
```

Comment:
Not needed for this PR but would be nice if we could search across all languages somehow.

Reply:
Added ability to search for English names if user has selected a different language:
<img width="447" alt="Screenshot 2025-03-17 at 23 42 02" src="https://github.com/user-attachments/assets/98ed4129-48ec-4d3d-bbcb-daf0353c144a" />